spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-lattice/pom.xml

```diff
@@ -27,6 +27,7 @@
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-lattice-core</artifactId>
 			<version>${spring-cloud-lattice.version}</version>
+			<scope>runtime</scope>
```

Comment:
I intend to add the lattice dependency that could bring in the `RedisConnectionFactory` for `AdminConfiguration` so that the `RedisMetricRepository` gets the cloud redis connectionfactory. But I understand that's what is required here. Perhaps, I should add `spring-cloud-lattice-connector` and `spring-cloud-spring-service-connector` to admin dependencies to get them working?